src/style.css

```diff
@@ -15,6 +15,16 @@ body {
   font-weight: bold;
 }
 
+#scrollableModal {
+  background: rgba(255, 228, 196, 0.8);
+  display: block;
```

Comment:
Display property is block by default, why did you use that here? Was it necessary?